post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -0,0 +1,60 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: multi-architecture-tuning-operator
+[id="multiarch-tuning-operator"]
+= Managing workloads on multi-architecture clusters by using the Multiarch Tuning Operator
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+[IMPORTANT]
+====
+The Multiarch Tuning Operator is a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration. 
+
+This Operator implements the `ClusterPodPlacementConfig` custom resource (CR) to support architecture-aware workload scheduling.
+
+When you create the `ClusterPodPlacementConfig` CR, the Multiarch Tuning Operator deploys an operand. This operand automates the configuration of scheduling predicates for the workloads based on their supported architectures. It uses the `kubernetes.io/arch` label set by the kubelet in the node objects.
+
+When a pod is created, the operand configures the `nodeAffinity` field in the pod's specification by computing the subset of architectures that the pod can support.
+
+Before a pod enters the scheduling cycle, the operand adds the `multiarch.openshift.io/scheduling-gate` scheduling gate that prevents the scheduling of the pod until the operand completes the following actions:
+
+. Compute a predicate for the `kubernetes.io/arch` label. 
+. Integrate the predicate as a `nodeAffinity` requirement in the pod's specification.
+. Remove the scheduling gate.
+
+This allows scheduling workloads only on nodes with supported architectures.
+
+[IMPORTANT]
+====
+The Technology Preview version of the Multiarch Tuning Operator does not support clusters with restricted network scenarios.
```

Comment:
```suggestion
The Technology Preview version of the Multiarch Tuning Operator does not support clusters with restricted network scenarios.
```
Does "with restricted network scenarios" mean only clusters running on restricted networks, or something broader?

Reply:
I think it means only clusters running on restricted networks. 

WDYT? @aleskandro

Reply:
it's fine to keep as is for me.